.github/workflows/release-31_promote-rc-to-final.yml

```diff
@@ -28,6 +28,10 @@ on:
         description: Tag matching the actual release candidate with the format polkadot-stableYYMM(-X)-rcX
         type: string
         required: true
+    outputs:
+      final_tag:
+        description: "Final release tag without RC suffix (e.g., polkadot-stableYYMM)"
+        value: ${{ jobs.promote-polkadot-rc-to-final.outputs.final_tag || jobs.promote-polkadot-parachain-rc-to-final.outputs.final_tag || jobs.promote-polkadot-omni-node-rc-to-final.outputs.final_tag || jobs.promote-frame-omni-bencher-rc-to-final.outputs.final_tag || jobs.promote-chain-spec-builder-rc-to-final.outputs.final_tag }}
```

Comment:
I assume this is so massive due to a matrix use in the downstream job? Maybe it would make sense then to move the tag creation to this job and just pass it to the downstream. What do you think?